dockerfiles/sandbox/start-with-nginx.sh

```
@@ -432,48 +411,32 @@ for i in $(seq 1 $NUM_WORKERS); do
-
-    PORT_OFFSET=$(( (retry_round + 1) * PORT_INCREMENT ))
-    echo "[$_H] Retry $((retry_round + 1)): ${#FAILED_WORKERS[@]} port conflicts, offset +$PORT_OFFSET"
-
-    for i in "${FAILED_WORKERS[@]}"; do
-        idx=$((i - 1))
-        new_port=$((SANDBOX_WORKER_BASE_PORT + i - 1 + PORT_OFFSET))
-        result=$(start_worker_fast $i $new_port)
-        WORKER_PIDS[$idx]="${result%%:*}"
-        ACTUAL_WORKER_PORTS[$idx]=$new_port
+    # Try up to MAX_STARTUP_RETRIES offsets if the port is occupied
+    for retry in $(seq 0 $MAX_STARTUP_RETRIES); do
```

Shellcheck (0.11.0)

[warning] retry appears unused. Verify use (or export if used externally).

(SC2034)